packages/api-specification/src/window/window-api.md

```diff
@@ -4,7 +4,7 @@ The Window API is used to create a new browser window. Compatability between sup
 
 ### Constructor
 ```javascript
-new ssf.Window(url, name, features)
+new ssf.Window(features[, callback, errorCallback])
```

Comment:
Can you please document the new callback arguments.

At this point it is probably worth thinking about when `callback` is actually invoked? can I send messages to the new window within this callback? Or do I have to wait for the `ready` lifecycle event to be called in the new window?

I don't think it's worth trying to normalise this behaviour just yet, but it is probably worth just discovering what the differences are and noting them.